libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -34,13 +34,14 @@ void harness()
      * MQTT_GetIncomingPacketTypeAndLength(). */
     NetworkContext_t networkContext;
 
-    __CPROVER_assume( networkContext != NULL );
-
     /* MQTT_GetIncomingPacketTypeAndLength() will set only the remainingLength
      * field in the input MQTTPacketInfo_t structure. */
-    MQTTPacketInfo_t * pIncomingPacketInfo = mallocCanFail( sizeof( MQTTPacketInfo_t ) );
+    MQTTPacketInfo_t * pIncomingPacket = NULL;
+
+    pIncomingPacket = allocateMqttPacketInfo( pIncomingPacket );
```

Comment:
Did you mean to to change the name from PacketInfo to Packet?  They seem like two different things.

Reply:
Yes because that is the how the parameter is named in the function. This information about the packet contains the actual packet contents in a pointer to a buffer it's fields.

Reply:
So, it has type of MQTTPacketInfo_t but we call it pIncomingPacket because the parameter has that name for MQTT_Get...()?  Is this autogenerated? Does it matter if they match?  Do we have inconsistencies in our API?

Reply:
All similiar functions in the API have the parameter name "pIncomingPacket" there doesn't appear to be an inconsistency. It doesn't really matter if they match, but I was just saying that this "pIncomingPacket" parameter name that was chosen I guess I can see the arguments for it's name.